src/ucs/arch/x86_64/cpu.c

```diff
@@ -737,27 +737,25 @@ ucs_status_t ucs_arch_get_cache_size(size_t *cache_sizes)
 }
 
 #if ((ENABLE_NT_BUFFER_TRANSFER) && (__AVX__))
-
-#define SWITCH_TO_NTSTORE_SZ (1464)
-
 static UCS_F_ALWAYS_INLINE
 size_t ucs_x86_nt_dst_buffer_transfer(void *dst, const void *src, size_t len,
-                                      int hint, size_t total_len)
+                                      unsigned int hint, size_t total_len)
 {
-    __m256i y0, y1, y2, y3;
     size_t offset;
+    __m256i y0, y1, y2, y3;
+    const size_t switch_to_nt_store_size = 1464;
```

Comment:
move if before other local vars since it's initialized

Reply:
Done.